### PR TITLE
Filipino Translation - 1 116  total translated words

### PR DESCRIPTION
**Pull request by mind-blast**
Completed Translation for the following two files 

replace.asc - 510 translated words
rerere.asc - 606 translated words

Note : code text and computer code text are not counted as translated words.

### CODE REVIEW

File: book/07-git-tools/sections/replace.asc [hunk 1]
```diff
@@ -89,18 +89,18 @@ $ echo 'get history from blah blah blah' | git commit-tree 9c68fdc^{tree}
...
 
-[NOTE]
+[TANDAAN]
 =====
-The `commit-tree` command is one of a set of commands that are commonly referred to as 'plumbing' commands.
-These are commands that are not generally meant to be used directly, but instead are used by *other* Git commands to do smaller jobs.
-On occasions when we're doing weirder things like this, they allow us to do really low-level things but are not meant for daily use.
+Ang `commit-tree` na utos ay isa sa isang set ng mga utos na karaniwang tinutukoy bilang 'plumbing' na mga utos.
+Ang mga utos na ito ay hindi sa pangkalahatang para sa gumagamit nang diretso, ngunit sa halip ay ginagamit ng *ibang* mga utos ng Git na gumawa ng maliliit na mga trabaho.
+Sa mga okasyon kapag kami ay gumawa ng mahiwang mga bagay na tulad nito, pinapahintulotan tayo na gumawa ng mababang antas ngunit hindi nakalaan para sa araw-araw na gamit.
```
Comment (heywens) [inline book/07-git-tools/sections/replace.asc]: `mahiwang` to `mahiwagang` o `kakaiba`

Comment (heywens) [inline book/07-git-tools/sections/replace.asc]: `araw-araw na gamit` to `pang-araw-araw na paggamit`

File: book/07-git-tools/sections/replace.asc [hunk 2]
```diff
@@ -112,11 +112,11 @@ Applying: fifth commit
 
 image::images/replace4.png[]
 
-OK, so now we've re-written our recent history on top of a throw away base commit that now has instructions in it on how to reconstitute the entire history if we wanted to.
-We can push that new history to a new project and now when people clone that repository, they will only see the most recent two commits and then a base commit with instructions.
+OK, kaya ngayon nagsusulat uli kami sa aming kamakailan lang na kasaysayan sa itaas ng isang itinapon na base commit na mayroon na ngayong mga tagubilin sa loob at kung papano i-reconstiture ang buong kasaysayan kung gusto nila.
```
Comment (heywens) [inline book/07-git-tools/sections/replace.asc]: `i-reconstiture` to `i-reconstitute`

File: book/07-git-tools/sections/replace.asc [hunk 3]
```diff
@@ -112,11 +112,11 @@ Applying: fifth commit
...
+Maaari nalang i-push ang bagong kasaysayan sa isang bagong proyelto at ngayon kapag ang tao ay nag-clone sa repositoryo na iyon, makikita nila ang pinakabagong dalawang commit at pagkatapos isang base commit na may mga pagtuturo.
```
Comment (heywens) [inline book/07-git-tools/sections/replace.asc]: `proyelto` to `proyekto`

File: book/07-git-tools/sections/replace.asc [hunk 4]
```diff
@@ -112,11 +112,11 @@ Applying: fifth commit
...
 
-Let's now switch roles to someone cloning the project for the first time who wants the entire history.
-To get the history data after cloning this truncated repository, one would have to add a second remote for the historical repository and fetch:
+Lumipat tayo ngayon ng mga papel para isang tao na mag-clone sa proyekto para sa unang pagkakataon na sino ang may gusto sa buong kasaysayan.
```
Comment (heywens) [inline book/07-git-tools/sections/replace.asc]: > para isang tao

to

> para sa isang tao


`na sino` to `kung sino`

File: book/07-git-tools/sections/replace.asc [hunk 5]
```diff
@@ -112,11 +112,11 @@ Applying: fifth commit
...
+Para makakuha ng kasaysayan ng datos pagkatapos mag-clone sa itong naka-truncate na repositoryo, magkakaroon ng isa upang madagdag ang pangalawang remote para sa makasaysayang repositoryo at kunin:
```
Comment (heywens) [inline book/07-git-tools/sections/replace.asc]: > pagkatapos mag-clone sa itong naka-truncate na repositoryo

to

> pagkatapos na mai-clone ito sa naka-truncate na repositoryo

File: book/07-git-tools/sections/replace.asc [hunk 6]
```diff
@@ -150,15 +150,15 @@ c6e1e95 fourth commit
 c1822cf first commit
 ----
 
-To combine them, you can simply call `git replace` with the commit you want to replace and then the commit you want to replace it with.
-So we want to replace the "fourth" commit in the master branch with the "fourth" commit in the `project-history/master` branch:
+Upang pagsamahin ang mga ito, maaari mong tawagan lang ang `git replace` na may commit na gusto mong ipalit at pagkatapos ang commit na gusto mong palitan ito sa.
+Kaya gusto naming palit ang "fourth" na commit sa master na branch na may "fourth" na commit sa `project-history/master` na branch:
```
Comment (heywens) [inline book/07-git-tools/sections/replace.asc]: `palit` to `palitan`

File: book/07-git-tools/sections/replace.asc [hunk 7]
```diff
@@ -150,15 +150,15 @@ c6e1e95 fourth commit
...
 
 [source,console]
 ----
 $ git replace 81a708d c6e1e95
 ----
 
-Now, if you look at the history of the `master` branch, it appears to look like this:
+Ngayon, kung titignan natin ang kasaysayan sa `master` na branch, lilitaw na ganito ang hitsura:
```
Comment (heywens) [inline book/07-git-tools/sections/replace.asc]: `titignan` to `titingnan`

File: book/07-git-tools/sections/replace.asc [hunk 8]
```diff
@@ -170,12 +170,12 @@ e146b5f fifth commit
 c1822cf first commit
 ----
 
-Cool, right?  Without having to change all the SHA-1s upstream, we were able to replace one commit in our history with an entirely different commit and all the normal tools (`bisect`, `blame`, etc) will work how we would expect them to.
+Cool, tama?  Nang walang pagbabago sa lahat sa SHA-1s na upstream, nagawa naming palitan ang isang commit sa aming kasaysayan na may isang ganap na naiiba na coomit at lahat ng mga normal na kasangkapan (`bisect`, `blame`, etc) gagana kung paano namin inaasahan ang mga ito.
```
Comment (heywens) [inline book/07-git-tools/sections/replace.asc]: `coomit` to `commit`

File: book/07-git-tools/sections/replace.asc [hunk 9]
```diff
@@ -170,12 +170,12 @@ e146b5f fifth commit
...
 
 image::images/replace5.png[]
 
-Interestingly, it still shows `81a708d` as the SHA-1, even though it's actually using the `c6e1e95` commit data that we replaced it with.
-Even if you run a command like `cat-file`, it will show you the replaced data:
+Kapansin-pansin, ito ay nagpapakita ng `81a708d` bilang ang SHA-1, kahit na ito talaga ang gumagamit ng `c6e1e95` na commit na datos na pinalit namin ito.
+Kahit na ikaw ay nagpatakbo ng isang utos tulad ng `cat-file`, ito ay nagpapakita sa iyo sa napalit na datos:
```
Comment (heywens) [inline book/07-git-tools/sections/replace.asc]: `napalit` to `napalitan`

File: book/07-git-tools/sections/replace.asc [hunk 10]
```diff
@@ -188,9 +188,9 @@ committer Scott Chacon <schacon@gmail.com> 1268712581 -0700
 fourth commit
 ----
 
-Remember that the actual parent of `81a708d` was our placeholder commit (`622e88e`), not `9c68fdce` as it states here.
+Tandaan na ang aktwal na magulang sa `81a708d` ay ang aming na commit (`622e88e`), hindi `9c68fdce` gaya ng sinasabi nito dito.
```
Comment (heywens) [inline book/07-git-tools/sections/replace.asc]: `aming na commit` to `aming placeholder na commit`

File: book/07-git-tools/sections/rerere.asc [hunk 11]
```diff
@@ -1,31 +1,31 @@
 [[_rerere]]
-=== Rerere
+=== Ang Rerere
 
-The `git rerere` functionality is a bit of a hidden feature.
-The name stands for ``reuse recorded resolution'' and, as the name implies, it allows you to ask Git to remember how you've resolved a hunk conflict so that the next time it sees the same conflict, Git can resolve it for you automatically.
+Ang `git rerere` pag-andar ay maliit na nakatagong tampok.
```
Comment (heywens) [inline book/07-git-tools/sections/rerere.asc]: > `git rerere` pag-andar

to 

> `git rerere` na pag-andar

File: book/07-git-tools/sections/rerere.asc [hunk 12]
```diff
@@ -1,31 +1,31 @@
...
+Ang pangalan ay nakatayong ``reuse recorded resolution'' at, bilang nagpapahiwatig na pangalan, pinapayagan ka nitong hilingan ang Git na tandaan kung paano mo nalutas ang isang malaking naputol na pagsasalungat kaya sa susunod na makakita ito ng parehong pagkasalungat, ang Git ay maaaring malutas ito para sa iyo ng awtomatiko.
```
Comment (heywens) [inline book/07-git-tools/sections/rerere.asc]: > The name stands for

could be

> Ang ibig sabihin ng pangalan ay ``reuse recorded resolution''

File: book/07-git-tools/sections/rerere.asc [hunk 13]
```diff
@@ -1,31 +1,31 @@
...
 
-There are a number of scenarios in which this functionality might be really handy.
-One of the examples that is mentioned in the documentation is when you want to make sure a long-lived topic branch will ultimately merge cleanly, but you don't want to have a bunch of intermediate merge commits cluttering up your commit history.
-With `rerere` enabled, you can attempt the occasional merge, resolve the conflicts, then back out of the merge.
-If you do this continuously, then the final merge should be easy because `rerere` can just do everything for you automatically.
+Mayroong mga bilang ng mga sitwasyon na kung saan ang pag-andar nito ay maaaring talagang magaling.
```
Comment (heywens) [inline book/07-git-tools/sections/rerere.asc]: `maaaring talagang` to `maaari talagang`

File: book/07-git-tools/sections/rerere.asc [hunk 14]
```diff
@@ -1,31 +1,31 @@
...
+Isa sa mga halimbawa na nabanggit sa dokumentasyon ay kapag gusto mo na tiyakin ang isang matagal na paksa na branch ay ganap na pagsasama nang malinis, ngunit ayaw mong makakaroon ng isang grupo ng intermediate merge na mga commit na nag-clutter pataas sa iyong kasaysayan ng commit.
```
Comment (heywens) [inline book/07-git-tools/sections/rerere.asc]: `makakaroon` to `magkakaroon`

File: book/07-git-tools/sections/rerere.asc [hunk 15]
```diff
@@ -1,31 +1,31 @@
...
+Sa `rerere` na pinagana, maaari kang magtangka sa minsang na-merge, malutas ang mga pagkasalungat, pagkatapos ay i-atras ang pag-merge.
```
Comment (heywens) [inline book/07-git-tools/sections/rerere.asc]: `malutas` to `lutasin` (command)

File: book/07-git-tools/sections/rerere.asc [hunk 16]
```diff
@@ -1,31 +1,31 @@
...
+Kung gagawin mo ito nang paulit-ulit, pagkatapos ay ang huling merge ay dapat na madali dahil sa `rerere` ay maaari lamang gawin ang lahat para sa iyo ng awtomatiko.
 
-This same tactic can be used if you want to keep a branch rebased so you don't have to deal with the same rebasing conflicts each time you do it.
-Or if you want to take a branch that you merged and fixed a bunch of conflicts and then decide to rebase it instead -- you likely won't have to do all the same conflicts again.
+Ang parehong taktika ay maaaring gamitin kung gusto mong itago ang isang branch na naka-rebase rebased kaya hindi mo kailangang harapin ang parehong pag-rebase sa mga sumasalungat sa bawat oras na ginawa mo ito.
+O kung gusto mong kumuha ng isang branch na  if you want to take a branch na nai-merge mo at nag-ayos ng maraming mga pagkakasalugat at pagkatapos ay magpasya na i-rebase ito sa halip -- malamang na hindi mo kailanganin gawin muli ang parehong mga pagkakasalungat muli.
```
Comment (heywens) [inline book/07-git-tools/sections/rerere.asc]: please remove the source phrase
~~if you want to take a branch~~

File: book/07-git-tools/sections/rerere.asc [hunk 17]
```diff
@@ -1,31 +1,31 @@
...
 
-Another application of `rerere` is where you merge a bunch of evolving topic branches together into a testable head occasionally, as the Git project itself often does.
-If the tests fail, you can rewind the merges and re-do them without the topic branch that made the tests fail without having to re-resolve the conflicts again.
+Isa pang application sa `rerere` na kung saan ikaw ay mag merge ng maraming nagbabago na paksang mga branch magkasama sa isang pagsubok sa ulo pangminsan-minsan, bilang ang Git na proyekto sa kanyang sarili ay madalas na ginagawa.
+Kung ang pagsubok ay nabigo, maaari mong i-rewind ang mga merge at gawin muli sila ng walang paksa sa branch na ginawa mong pagsubok na walang nareresulba na mga pagsasalungat muli.
 
-To enable `rerere` functionality, you simply have to run this config setting:
+Para mapagana ang `rerere` na pag-andar, kailangan mo lang na ipagtakbo ang config na setting na ito:
```
Comment (heywens) [inline book/07-git-tools/sections/rerere.asc]: `ipagtakbo` to `ipatakbo` or `patakbuhin`

File: book/07-git-tools/sections/rerere.asc [hunk 18]
```diff
@@ -1,31 +1,31 @@
...
 $ git config --global rerere.enabled true
 ----
 
-You can also turn it on by creating the `.git/rr-cache` directory in a specific repository, but the config setting is clearer and enables that feature globally for you.
+Maaari mo ring buksan ito sa pamamagitan ng paglikha ng `.git/rr-cache` na direktoryo sa isang tiyak na repositoryo, ngunit ang config na setting ay mas malinaw at nagpapagana sa pag-andar sa buong mundo para sa iyo.
 
-Now let's see a simple example, similar to our previous one.
-Let's say we have a file named `hello.rb` that looks like this:
+Ngayon tingnan natin ang isang simpleng halimbawa, katulad sa ating nakaraang isa.
+Sabihin natin na meron tayong isang file na pangalan ay `hello.rb` na ganito ang hitsura:
```
Comment (heywens) [inline book/07-git-tools/sections/rerere.asc]: > isang file na pangalan ay `hello.rb`

could be translated to:

> isang file na pinangalanang `hello.rb`

File: book/07-git-tools/sections/rerere.asc [hunk 19]
```diff
@@ -99,7 +99,7 @@ $ git rerere diff
  end
 ----
 
-Also (and this isn't really related to `rerere`), you can use `git ls-files -u` to see the conflicted files and the before, left and right versions:
+Gayundin (at ito ay hindi talagang may kaugnayan sa `rerere`), maaari mong gamitin ang `git ls-files -u` para tingnan ang nagkasalungat na mga file at ang bago, ang kaliwa at ang kanan na mga bersyon:
```
Comment (heywens) [inline book/07-git-tools/sections/rerere.asc]: ang bago, ~~ang~~ kaliwa at ~~ang~~ kanan na mga bersyon

File: book/07-git-tools/sections/rerere.asc [hunk 20]
```diff
@@ -129,9 +129,9 @@ $ git rerere diff
  end
 ----
 
-So that basically says, when Git sees a hunk conflict in a `hello.rb` file that has ``hello mundo'' on one side and ``hola world'' on the other, it will resolve it to ``hola mundo''.
+Kaya talagang sabi, kapag ang Git ay nakakita ng maraming pagkakasalungat sa isang `hello.rb` na file na mayroong ``hello mundo'' sa ibang panig at ang ``hola world'' ay sa kabila, ito ay maglulutas sa ``hola mundo''.
 
-Now we can mark it as resolved and commit it:
+Ngayon ay markahan natin ito bilang natutas at na-commit ito:
```
Comment (heywens) [inline book/07-git-tools/sections/rerere.asc]: `natutas` to `nalutas`

File: book/07-git-tools/sections/rerere.asc [hunk 21]
```diff
@@ -207,7 +207,7 @@ index a440db6,54336ba..0000000
 
 image::images/rerere3.png[]
 
-You can also recreate the conflicted file state with `git checkout`:
+Maaari ka ding lumikha sa nagkasalungay na file na estado gamit ang `git checkout`:
```
Comment (heywens) [inline book/07-git-tools/sections/rerere.asc]: `nagkasalungay` to `nagkasalungat`

File: book/07-git-tools/sections/rerere.asc [hunk 22]
```diff
@@ -239,8 +239,8 @@ def hello
 end
 ----
 
-We have re-resolved the file automatically using the `rerere` cached resolution.
-You can now add and continue the rebase to complete it.
+Meron kaming nalutas na file na awtomatikong gumagamit ng `rerere` na cached na resolusyon.
+Maaari mo na ngayong idagdag at ipagpatuloy ang pagrebase upang tapusin ito.
```
Comment (heywens) [inline book/07-git-tools/sections/rerere.asc]: `pagrebase` to `pag-rebase`

File: book/07-git-tools/sections/rerere.asc [hunk 23]
```diff
@@ -249,4 +249,4 @@ $ git rebase --continue
 Applying: i18n one word
 ----
 
-So, if you do a lot of re-merges, or want to keep a topic branch up to date with your master branch without a ton of merges, or you rebase often, you can turn on `rerere` to help your life out a bit.
+Kaya, kung ikaw ay maraming gagawin na mga pag-re-remerge, o nais na panatilihin ang isang paksa na branch na bagongbago sa maraming mga merge, o madalas mong i-rebase, maaari mong i-on ang `rerere` upang tulungan ang iyong bugay nang kaunti.
```
Comment (heywens) [inline book/07-git-tools/sections/rerere.asc]: * `re-merges` could be translated to `pag-remerge`
* `bagongbago` to `bagong-bago`
* `bugay` to `buhay`

**Review by heywens — COMMENTED** [review]
Hi Sir @robin-ho, I'm done reviewing your translation. Kindly check my feedback that require some changes.

Thanks!

File: book/07-git-tools/sections/replace.asc (hunk 1 above)
Comment (mind-blast) [reply book/07-git-tools/sections/replace.asc]: done change

Comment (mind-blast) [reply book/07-git-tools/sections/replace.asc]: done change

File: book/07-git-tools/sections/replace.asc (hunk 2 above)
Comment (mind-blast) [reply book/07-git-tools/sections/replace.asc]: done change

File: book/07-git-tools/sections/replace.asc (hunk 3 above)
Comment (mind-blast) [reply book/07-git-tools/sections/replace.asc]: done change

File: book/07-git-tools/sections/replace.asc (hunk 4 above)
Comment (mind-blast) [reply book/07-git-tools/sections/replace.asc]: done change

File: book/07-git-tools/sections/replace.asc (hunk 5 above)
Comment (mind-blast) [reply book/07-git-tools/sections/replace.asc]: done change

File: book/07-git-tools/sections/replace.asc (hunk 6 above)
Comment (mind-blast) [reply book/07-git-tools/sections/replace.asc]: done change

File: book/07-git-tools/sections/replace.asc (hunk 7 above)
Comment (mind-blast) [reply book/07-git-tools/sections/replace.asc]: done change

File: book/07-git-tools/sections/replace.asc (hunk 8 above)
Comment (mind-blast) [reply book/07-git-tools/sections/replace.asc]: done change

File: book/07-git-tools/sections/replace.asc (hunk 9 above)
Comment (mind-blast) [reply book/07-git-tools/sections/replace.asc]: done change

File: book/07-git-tools/sections/replace.asc (hunk 10 above)
Comment (mind-blast) [reply book/07-git-tools/sections/replace.asc]: done change

File: book/07-git-tools/sections/rerere.asc (hunk 11 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rerere.asc]: done change

File: book/07-git-tools/sections/rerere.asc (hunk 12 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rerere.asc]: done change

File: book/07-git-tools/sections/rerere.asc (hunk 13 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rerere.asc]: done change

File: book/07-git-tools/sections/rerere.asc (hunk 14 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rerere.asc]: done change

File: book/07-git-tools/sections/rerere.asc (hunk 15 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rerere.asc]: done change

File: book/07-git-tools/sections/rerere.asc (hunk 16 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rerere.asc]: done change

File: book/07-git-tools/sections/rerere.asc (hunk 17 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rerere.asc]: done change

File: book/07-git-tools/sections/rerere.asc (hunk 18 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rerere.asc]: done change

File: book/07-git-tools/sections/rerere.asc (hunk 19 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rerere.asc]: done change

File: book/07-git-tools/sections/rerere.asc (hunk 20 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rerere.asc]: done change

File: book/07-git-tools/sections/rerere.asc (hunk 21 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rerere.asc]: done change

File: book/07-git-tools/sections/rerere.asc (hunk 22 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rerere.asc]: done change

File: book/07-git-tools/sections/rerere.asc (hunk 23 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rerere.asc]: done change

**Comment by mind-blast** [comment]
Change all suggested words.. Thanks for the proof read.

**Pull request merged**